openedx/core/djangoapps/user_api/preferences/api.py

```diff
@@ -421,8 +422,8 @@ def _create_preference_update_error(preference_key, preference_value, error):
 
 def get_country_time_zones(country_code=None):
     """
-    Returns a list of time zones commonly used in given country
-    or list of all time zones, if country code is None.
+    Returns a sorted list of time zones commonly used in given
+    country or list of all time zones, if country code is None.
```

Comment:
I'm just using the country_timezones() function from the pytz library. Unfortunately,  they don't give much info: http://pytz.sourceforge.net/#country-information